specification/entities.yaml

```diff
@@ -117,6 +117,8 @@ components:
           description: >
             key is a field oddrn.
             For example: "//aws/glue/{account_id}/{database}/{tablename}/columns/{column_name}"
+        custom_stats:
+          $ref: '#/components/schemas/CustomFieldStat'
```

Comment:
rename without field

Reply:
it's named like all other stats: ComplexFieldStat, BooleanFieldStat etc. If I rename it it will be different from all other statistics